modules/upload-definitions-to-central-admin-pass.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * configuration/enable-offline-mode.adoc
+:_module-type: PROCEDURE
+[id="upload-definitions-to-central-admin-pass_{context}"]
+= Uploading definitions to Central by using the administrator password
+
+You can upload the vulnerability definitions database (that Scanner uses) to Central by using your {product-title} administrator password.
```

Comment:
I think you could get rid of the parentheses here too. I don't think the IBM style guide likes them in general, so it's best to try to remove if you can (below module too)